default/python/AI/AIDependencies.py

```diff
@@ -668,7 +672,13 @@
         REPAIR_PER_TURN: 2,
     },
     "SH_SPATIAL_FLUX": {
-        STEALTH_MODIFIER: -30,
+        STEALTH_MODIFIER: [
```

Comment:
I don't understand what does it mean.  Looks like error and unnecessary code because it always 10. 

This should be easy to understand without any comments.  Please don't add magic to code. 

Also I suggest to do next steps:
- move parsing to separate module
- make tests (freeorion\default\python\tests)

After what we can back to review.

PS. We already have some new language in research AI:  it works on function.  Maybe we can reuse it?

https://github.com/freeorion/freeorion/blob/master/default/python/AI/ResearchAI.py#L356